lizrd/support/logging.py

```diff
@@ -195,6 +221,7 @@ def report_scalar(
         value: float,
         iteration: int,
         series: Optional[str] = None,
+        token_scale: bool = False,
```

Comment:
Why we can't just log tokens for every run?

Reply:
Ok, I'm gonna make this default

Reply:
Ok sorry, I this should be false, because by default we don't want ANY metric to be in token scale, only loss dependent ones, which are directly specified.

Reply:
renamed to processed_token_scale